org.lyx.LyX.yml

```
@@ -11,19 +11,32 @@ finish-args:
   - --device=dri
   - --share=ipc
   - --persist=.lyx
+  - --persist=Rpkgs
   - --filesystem=xdg-pictures:ro
   - --filesystem=xdg-documents
-  - --env=PATH=/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
+  - --env=PATH=/app/Extensions/R/bin:/app/Extensions/LilyPond/bin:/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
 add-extensions:
   org.freedesktop.Sdk.Extension.texlive:
     version: "23.08"
     directory: texlive
     autodelete: false
+  org.lyx.LyX.LilyPond:
+    directory: Extensions/LilyPond
```

IMHO it should be `lilypond` to be consistent
```suggestion
    directory: lilyPond
```

---

or you can make it "subdir" so that there is only one extension point defined.

---

I will try it